src/ert/config/queue_config.py

```diff
@@ -108,11 +115,68 @@ def generate_dict(option_list: List[Tuple[str, str]]) -> Dict[str, List[str]]:
 
 queue_memory_options: Mapping[str, List[str]] = {
     "LSF": [],
-    "SLURM": [],
+    "SLURM": ["MEMORY_PER_CPU", "MEMORY"],
     "TORQUE": ["MEMORY_PER_JOB"],
     "LOCAL": [],
 }
 
+
+@dataclass
+class QueueMemoryStringFormat:
+    suffixes: List[str]
+    parse: Callable[[str], str]
+
+    def validate(self, mem_str_format: str):
+        return len([s for s in self.suffixes if mem_str_format.endswith(s)]) > 0
+
+
+def parse_lsf_memopt(s: str):
+    return s.upper()
+
+
+def parse_slurm_memopt(s: str):
+    return s.lower().replace("b", "").upper()
+
+
+def parse_torque_memopt(s: str):
+    if re.match(r"\d+[kgmt](?!\w)", s, re.IGNORECASE):
+        return s.lower() + "b"
+    if re.match(r"^\d+$", s):
+        return s + "mb"
+
+    return s
+
+
+queue_memory_usage_formats: Mapping[str, QueueMemoryStringFormat] = {
+    "LSF": QueueMemoryStringFormat(
+        suffixes=[
+            "",
```

Comment:
Using `bsub` directly, I can only get `rusage` to work without any units, and AFAICT we pass the `LSF_RESOURCE` text verbatim to `bsub -R`... so I'm not sure we can pass any units at all.

Reply:
That is a bit odd, are we using an older version maybe? Official LSF docs on resource req strings (https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=requirements-resource-requirement-strings) give these examples: `bsub -C 500MB -M 1G -S 1TB -F 1GB -R "rusage[mem=512MB:swp=1GB:tmp=1TB]" sleep 100`. From the snake oil test config we have `"select[x86_64Linux] same[type:model]"` as value for the `LSF_RESOURCE`, so I'm not sure LSF_RESOURCE is even a pure memory specification keyword...

Reply:
TLDR; Omitting "parsing" of the LSF stuff as part of this, not finding any direct ways to specify mem usage on LSF other than as part of the resource string